www/view/Main.view.xml

```diff
@@ -1,9 +1,84 @@
-<mvc:View controllerName="SAPUI5ExampleSAPUI5Example.controller.Main" xmlns:html="http://www.w3.org/1999/xhtml" xmlns:mvc="sap.ui.core.mvc"
+<mvc:View controllerName="SAPUI5ExampleSAPUI5Example.controller.Main" 
+xmlns:html="http://www.w3.org/1999/xhtml" 
+xmlns:mvc="sap.ui.core.mvc"
+xmlns:core="sap.ui.core"
 	displayBlock="true" xmlns="sap.m">
 	<App>
 		<pages>
 			<Page title="{i18n>title}">
-				<content></content>
+				<content>
+				   <Table
+				       id="table"			    
+				       headerText="{i18n>tableTitle}"
+				       items="{operators>/Operators}"
+                       >
+                       <columns>
+                           <Column mergeDuplicates="true">
+                             <header>
+                                <Text text="{i18n>tableColumnName}"/>
+                              </header>
+                           </Column>
+                           <Column mergeDuplicates="true">
+                             <header>
+                                <Text text="{i18n>tableColumnStatus}"/>
+                              </header>
+                           </Column>
+                           <Column mergeDuplicates="true">
+                             <header>
+                                <Text text="{i18n>tableColumnPlant}"/>
+                              </header>
+                           </Column>
+                           <Column mergeDuplicates="true">
+                             <header>
+                                <Text text="{i18n>tableColumnPhone}"/>
+                              </header>
+                           </Column>
+                            <Column mergeDuplicates="true">
+                             <header>
+                                <Text text="{i18n>tableColumnEmail}"/>
+                              </header>
+                           </Column>
+                           
+                       </columns>
+                         <ColumnListItem >
+                              <Text text="{operators>name}" />
+                              <Text text="{operators>status}" />
+                              <Text text="{operators>plant}" />
+                              <Text text="{operators>phone}" />
+                              <Text text="{operators>email}" />
+                         
+                         </ColumnListItem>
+				   </Table>
+				
+				</content>
+				
+				<footer>
+				    <Bar>
+				      <contentRight>
+				           <Text text="{i18n>GroupTitleOrder}"></Text>
+					       <Select
+					          id="groupSelect"
+						      change="handleGroup"
+						      icon="sap-icon://sort-ascending"
+						      type="IconOnly"
+						      selectedKey="None"
+						      autoAdjustWidth="true" >
+						      <core:Item
+							    key="NoneKey"
```

Comment:
Try to make a model of these keys. Disregard this as a necessary correction, just a note.